.github/test_sdk/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(test_sdk)
+
+cmake_minimum_required(VERSION 3.2)
+
+find_package(f3d REQUIRED)
+
+add_executable(test_sdk main.cxx)
```

Comment:
could this be part of the test suite instead ?

Reply:
No

Reply:
we will probably needs more tests of f3dlib directly. But we will adapt at that time.